net-misc/kea/kea-9999.ebuild

```diff
@@ -1,149 +0,0 @@
-# Copyright 1999-2025 Gentoo Authors
```

Comment:
FWIW, I'm fine with dropping it (as it clearly was in a bad state), but Peter, if you want to bring this back later, that may well be useful (especially in preparing for a Meson port).